test/constants.js

```diff
@@ -12,74 +12,52 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import constants from '../src/constants';
-
-describe('constants module', () => {
-	it('should have a fixedPoint integer', () => {
-		const { fixedPoint } = constants;
-		(fixedPoint).should.be.an.integer();
+import {
+	fixedPoint,
+	dappFee,
+	delegateFee,
+	inTransferFee,
+	outTransferFee,
+	multisignatureFee,
+	signatureFee,
+	sendFee,
+	voteFee,
+} from '../src/constants';
+
+describe('constants', () => {
+	it('should know the fixedPoint', () => {
+		(fixedPoint).should.be.equal(1e8);
```

Comment:
Testing the shape of the export and types of properties makes sense, but we shouldn't test constants.